_posts/en/newsletters/2023-08-02-newsletter.md

```diff
@@ -0,0 +1,173 @@
+    can force close the channel to attempt to receive a refund of the
+    amount she forwarded to Bob, and her software may automatically do
+    this to prevent Alice from potentially losing the money to a node
+    upstream of her.  But, if she force closes the channel, she
+    might end up in the same position as Bob: she's either unable to
+    claim the refund or doesn't attempt it because it's not economical.
+    That means a useful channel between Alice and Bob was closed for no
+    gain to either one of them.  This problem could be repeated over and
+    over again for any hops upstream of Alice, resulting in a cascade of
+    unwanted channel closures.
+
+    The solution implemented in this PR is for Bob to wait as long as
+    reasonable to claim a refund and, if it's not going to happen, he
+    creates an offchain cancellation of the HTLC he received from Alice,
+    allowing their channel to continue operating even if it means he
+    might lose the amount of the HTLC.
+
+- [Core Lightning #6399][] adds support to the `pay` command for paying
+  invoices created by the local node.  This can simplify account
+  management code for software that calls CLN in the background, as
+  discussed in a recent [mailing list thread][fiatjaf custodial].
+
+- [Core Lightning #6389][] adds an optional CLNRest service, "a
+  lightweight Python-based core lightning plugin that transforms RPC
+  calls into a REST service. By generating REST API endpoints, it
+  enables the execution of Core Lightning's RPC methods behind the
+  scenes and provides responses in JSON format."  See its
+  [documentation][clnrest doc] for details.
+
+- [Core Lightning #6403][] and [#6437][core lightning #6437] move the
+  runes authorization and authentication mechanism out of CLN's commando
+  plugin (see [Newsletter #210][news210 commando]) and into its core
+  functionality, allowing other plugins to use them.  A number of
+  commands related to creating, destroying, and renaming runes are also
+  updated.
+
+- [Core Lightning #6398][] extends the `setchannel` RPC with a new
+  `ignorefeelimits` option that will ignore the minimum onchain fee
+  limits for the channel, allowing the remote channel counterparty to
+  set a feerate below the minimum the local node will permit.  This can
+  help work around a potential bug in another LN node implementation or
+  can be used to eliminate fee contention as a source of problems in
+  partly trusted channels.
+
+- [Core Lightning #5492][] adds User-level Statically Defined Tracepoints
+  (USDT) and the means to use them.  These allow users to probe the
+  internal operation of their node for debugging without introducing any
+  significant overhead when tracepoints aren't being used.  See
+  [Newsletter #133][news133 usdt] for the previous inclusion of USDT
+  support into Bitcoin Core.
+
+- [Eclair #2680][] adds support for the quiescence negotiation protocol
```

Comment:
Perhaps one more sentence on what a “quiescence negotiation protocol” might be would be helpful.